new/rfc-meetbot.md

```diff
@@ -0,0 +1,65 @@
+---
+RFC: unassigned
+Author: JJ Asghar <jj@chef.io>
+Status: Draft
+Type: Process
+---
+
+# Enabling meetbot for Chef IRC Community meetings.
+
+I would like to enable [meetbot][meetbot] for the Chef IRC Community Meetings.
+The ability to automatically take notes and executive summaries can open up
+people to have conversations instead of the dogma of making sure the conversations
+stay on track.
```

Comment:
From IRC conversation, I think you mean something like:

Shifting responsibility for taking meeting notes from the designated secretary to software allows the person acting as the secretary to instead participate fully in the discussion.